### ISSUE TITLE
waveshare35a on Kernel 5.x on arm64

### ISSUE THREAD

**Issue by satmandu** [swkim01/waveshare-dtoverlays#24]
Upgraded from ubuntu cosmic to disco and the display stopped working.

Has anybody else had any experiences with the dtoverlays failing with 5.x? Or with Disco?

**Comment by satmandu** [comment]
Apparently on disco fb is at /dev/fb2

**Comment by satmandu** [comment]
This issue due to commits starting in 5.1, which were finally fixed in 5.4, as per a dicussion with Phil Reid, one of the authors of the relevant patches fixing fbtft:

```
As the driver is pretty flexible in how it talks to displays there's
a number of things that could be missing.

Perhaps check the polarity of the gpio pins.
Prior to the switch to gpiod polarity definitions
in the dts file where not being honoured.

That'd be these two lines in your dts I think.

        reset-gpios = <&gpio 25 0>;
        dc-gpios = <&gpio 24 0>;

Check 3rd parameter.
Quick check of the ili9486 shows that it is active low reset.
So try:

        reset-gpios = <&gpio 25 GPIO_ACTIVE_LOW>;
```
```
GPIO_ACTIVE_LOW is a macro defined in the linux dt-bindings header

https://github.com/torvalds/linux/tree/master/include/dt-bindings
https://github.com/torvalds/linux/blob/master/include/dt-bindings/gpio/gpio.h

Don't think dtc directly does the #includes, so you've got to run it all thru the pre-processor first.
```

(And later in the conversation:)
```
So replacing GPIO_ACTIVE_LOW with "1" should also suffice, and allow me to drop needing an include file in the dts file 
```

Note that this may need to be done for other fbtft drivers.

Also referred to here: https://github.com/notro/fbtft/issues/546
